internal/command/command_run.go

```diff
@@ -73,6 +73,14 @@ func DetermineImage(ctx context.Context, appName string, imageOrPath string) (im
 		}
 		opts.BuildArgs = extraArgs
 
+		if cfg != nil && cfg.Experimental != nil {
+			opts.UseZstd = cfg.Experimental.UseZstd
+		}
+		// use-zstd passed through flags takes precedence over the one set in config
+		if useZstd := flag.GetBool(ctx, "use-zstd"); useZstd {
```

Comment:
you can use `flag.IsSpecificied(ctx, "use-zstd")` and then `opts.UseZstd = flag.GetBool(ctx, "use-zstd")` to override a true fly.toml value with `--use-zstd=false` cli flag.

Reply:
addessed in https://github.com/superfly/flyctl/pull/4065/commits/8f0f5e5c549ef9b4954692f2c51335afdba8dfca